commands/randfuture.js

```diff
@@ -0,0 +1,60 @@
+const factory = require('../lib/factory')
+const logger = require('../lib/logger')
+const moment = require('moment')
+const randSched = require('../lib/random-schedule')
+
+exports.command = 'randfuture <extension> <resources>'
+
+exports.describe = 'starts some chaos at a random point in the next month\nNB: there are no time window contraints, can be anytime in a 24 hour period'
+
+exports.builder = {
+  extension: {
+    description: 'the name of the extension',
+    required: true
+  },
+  resources: {
+    description: 'the resources (see `chaos resgen`)',
+    required: true
+  }
+}
+
+exports.handler = (argv) => {
+  const authenticator = factory.AzureAuthenticator.create()
+  const registry = factory.ExtensionRegistry.create()
+
+  const asyncAuthProvider = argv.accessToken ? Promise.resolve(argv.accessToken) : authenticator.interactive()
+
+  var secondsInterval = randSched.calcRandomSecond(1, 2638000)
```

Comment:
what is this magic number?

Reply:
I sometimes see these things written out as `60 * 60 * 24 * 31` (60 seconds * 60 minutes * 24 hours * 31 days) so devs better understand what it is